pylint/lint/pylinter.py

```diff
@@ -228,6 +228,7 @@ def make_options() -> Tuple[Tuple[str, OptionDict], ...]:
                     "type": "csv",
                     "metavar": "<file>[,<file>...]",
                     "dest": "black_list",
+                    "kwargs": {"old_names": ["black_list"]},
```

Comment:
Does it means that the old parameter will be ignored ?

Reply:
No I have set it up so that both `dest` and `old_names` in `kwargs` will also get the value of the option. So both `namespace.ignore` as well as `namespace.black_list` will work.
I have added both `dest` and `old_names` here for test coverage as the order of the `ifs` makes it that this together with the `black_list_re` option tests all combinations 😄